openlibrary/core/bookshelves.py

```diff
@@ -255,3 +255,14 @@ def user_with_most_books(cls):
                  "ORDER BY counted DESC, username LIMIT 100")
         result = oldb.query(query)
         return list(result)
+
+    @classmethod
+    def migrate_bookshelves(cls, from_work_id: int, to_work_id: int):
```

Comment:
The method returns a list of what?
```suggestion
    def migrate_bookshelves(cls, from_work_id: int, to_work_id: int) -> list:
```